src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/card/card.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'cd-card',
+  templateUrl: './card.component.html',
+  styleUrls: ['./card.component.scss']
+})
+export class CardComponent {
+  @Input()
+  cardTitle: string;
```

Comment:
Although `title` is a common HTML attribute, it's not a bad thing, since in fact it's a semantic one and some screen-readers could interpret it.